ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -69,6 +70,58 @@ do
   scp "${SSHOPTS[@]}" "${item}" "root@${IP}:manifests/${manifest##manifest_}"
 done <   <( find "${SHARED_DIR}" \( -name "manifest_*.yml" -o -name "manifest_*.yaml" \) -print0)
 
+
+# dev-scripts setup for baremetal clusters
+CIRFILE=$SHARED_DIR/cir
+EXTRAFILE=$SHARED_DIR/cir-extra
+BMJSON=$SHARED_DIR/bm.json
+if [ -e "$CIRFILE" ] && [ "$(cat $CIRFILE | jq -r .type)" == "cluster" ] ; then
```

Comment:
To help managing the complexity of this step, it would be nice to refactor out this block into its own (documented) function